packages/react-scripts/template/babelrc

```diff
@@ -0,0 +1,12 @@
+{
+  "plugins": [
+    ["module-resolver", {
+      "root": ["./src"],
+      "alias": {
+        "components": "./src/app/components",
+        "screens": "./src/app/screens",
+        "store": "./src/store"
```

Comment:
We don't have a `store` folder. Maybe you meant `src/redux`?

Reply:
It's what we defined, I'd call it reducers to be honest.

I think we don't need to define it here, call it `redux` and we can discuss the folder structure outside this pull request.